src/backend/core/tests/test_models_items.py

```diff
@@ -639,6 +640,39 @@ def test_models_items__email_invitation__failed(mock_logger, _mock_send_mail):
     assert isinstance(exception, smtplib.SMTPException)
 
 
+@mock.patch("core.models.send_mail")
+@mock.patch.object(Logger, "debug")
```

Comment:
You can use the `caplog` fixture instead of mocking the logger. It is made for that purpose. https://docs.pytest.org/en/stable/how-to/logging.html